pkgs/desktops/gnome-3/extensions/caffeine/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, glib, gettext, bash }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-extension-caffeine-${version}";
+  version = "ce0d0d4d3a9fed4b35b82cf59609a00502862271";
```

Comment:
We prefer `unstable-<year>-<month>-<day>` as version number for projects without releases over git hashes, as the first scheme is sortable, when updating packages with `nix-env`.